source/website/src/components/VideoPlayer.vue

```diff
@@ -77,8 +77,9 @@ export default {
         },
         markers: []
       });
-      this.player.autoplay('muted');
-      this.player.currentTime(0);
+      // Set player time to 2 second mark in order to avoid a black poster image for
```

Comment:
I'm including this small change in this PR even though it's unrelated to the topic of the PR. This is such a small change that I don't think it needs a separate PR.